service/src/main/java/com/fecklessweasel/service/UserServlet.java

```diff
@@ -0,0 +1,84 @@
+package com.fecklessweasel.service;
+
```

Comment:
I don't know if there's a plugin to handle this automatically, but it might be nice to order our imports alphabetically or something.